community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/NodeRecord.java

```diff
@@ -30,22 +30,35 @@
 
 public class NodeRecord extends PrimitiveRecord
 {
-    private long nextRel = Record.NO_NEXT_RELATIONSHIP.intValue();
-    private long labels = Record.NO_LABELS_FIELD.intValue();
-    private Collection<DynamicRecord> dynamicLabelRecords = emptyList();
-    private boolean isLight = true;
+    private long nextRel;
+    private long labels;
+    private Collection<DynamicRecord> dynamicLabelRecords;
+    private boolean isLight;
     private boolean dense;
 
     public NodeRecord( long id )
     {
-        super( id, Record.NO_NEXT_PROPERTY.intValue() );
+        super( id );
     }
 
+    public NodeRecord initialize( boolean inUse, long nextProp, boolean dense, long nextRel, long labels )
+    {
+        super.initialize( inUse, nextProp );
+        this.nextRel = nextRel;
+        this.dense = dense;
+        this.labels = labels;
+        this.dynamicLabelRecords = emptyList();
+        this.isLight = true;
+        return this;
+    }
+
+    @Deprecated
```

Comment:
Remove/replace over time. Removing all those constructors in this PR would inflate it massively due to Consistency checker tests using those.